modules/core/src/main/java/org/apache/ignite/internal/util/StripedExecutor.java

```diff
@@ -48,15 +48,20 @@
 import org.apache.ignite.thread.IgniteThread;
 import org.jetbrains.annotations.NotNull;
 
+import static org.apache.ignite.IgniteSystemProperties.IGNITE_STRIPED_POOL_STARVATION_THRESHOLD;
+
 /**
  * Striped executor.
  */
 public class StripedExecutor implements ExecutorService {
+    /** Default threshold for starvation check */
+    private static final long DEFAULT_THRESHOLD = 30 * 1000;
```

Comment:
Why not use Failure Detection Timeout? We're doing this for all other thread blocking detection.

Reply:
Yeah, sure, good idea, I'll fix.